core/src/main/java/uk/gov/gchq/koryphe/serialisation/json/SimpleClassKeySerializer.java

```diff
@@ -29,9 +29,13 @@
  * for {@link Class}es as Map keys, which utilises the {@link SimpleClassNameIdResolver} to
  * allow simple class names to be used.
  */
-public class SimpleClassKeySerializer extends StdKeySerializer {
+public class SimpleClassKeySerializer extends StdKeySerializers.Default {
     private static final long serialVersionUID = -1765103660342721103L;
 
+    public SimpleClassKeySerializer() {
+        super(3, Class.class);
```

Comment:
typeId = 3 ? can you update me?

Reply:
There's Javadoc missing here - the parent class defines some package private constants to specify the typeId. In this case: `final static int TYPE_CLASS = 3;`